ports/plunderislands/Plunder Islands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/plunderislands/
+CONFDIR="$GAMEDIR/conf/"
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_DATA_HOME="$CONFDIR"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+cd $GAMEDIR
+
+# patch game
+[ -f "./gamedata/Plunder Islands.pck" ] && $controlfolder/xdelta3 -d -s "gamedata/Plunder Islands.pck" "patch/plunderislandspatch.xdelta" "gamedata/plunderislandspatched.pck" && rm "gamedata/Plunder Islands.pck"
+
+runtime="frt_3.5.2"
+if [ ! -f "$controlfolder/libs/${runtime}.squashfs" ]; then
+  # Check for runtime if not downloaded via PM
+  if [ ! -f "$controlfolder/harbourmaster" ]; then
+    echo "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info." > /dev/tty0
```

Comment:
As suggested by tabreturn:
```suggestion
    pm_message "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info."
```

Also you can do this 
```shell
    echo "This port requires the latest version of PortMaster."
    text_viewer -e -f 25 -w -t "PortMaster needs to be updated" -m "This port requires the latest version of PortMaster. Please update PortMaster first.\n\nPress SELECT to close this window."
    exit 0
```

but you'll need to add `text_viewer` binary to your port and in the PATH. See [here](https://github.com/PortsMaster-MV/PortMaster-MV-New/blob/1ad33356c0f94a7949a4ad46eb670d3387e7078c/ports/render96ex/render96ex.sh#L107) for an example.